views/Study.react.js

```diff
@@ -153,9 +153,12 @@ class Study extends Component {
       let completed = (matrixDataList[item].project in this.state.complete && this.state.complete[matrixDataList[item].project].indexOf(matrixDataList[item].subject) > -1);
       if (!completed) {
         numOfDays = diffDates(consentDateObject, this.state.todayDateObject);
+      }
```

Comment:
Suppose today is 28th February. Then, the study-level graphs should show Feb 1st as the earliest day. However, if a subject consented only on 15th, their graph should show 15th as the earliest day. Hence, line 160 has been pushed out of the above if block.

More explanation is at issue https://github.com/AMP-SCZ/dpdash/issues/56